prod_api/content_types/service.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2025 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from ..service import ProdApiService
+import bson
+import re
+import superdesk
+
+
+class ContentTypesService(ProdApiService):
```

Comment:
I wonder why it's extending that base service?

Reply:
to use predefined functions like `find_one`